src/main/java/com/powsybl/openloadflow/ac/equations/ClosedBranchSide2ReactiveFlowEquationTerm.java

```diff
@@ -29,8 +29,11 @@
 
     private double dq2dph2;
 
-    public ClosedBranchSide2ReactiveFlowEquationTerm(LfBranch branch, LfBus bus1, LfBus bus2, VariableSet variableSet) {
-        super(branch, bus1, bus2, variableSet);
+    private double dq2da1;
+
+    public ClosedBranchSide2ReactiveFlowEquationTerm(LfBranch branch, LfBus bus1, LfBus bus2, VariableSet variableSet,
+                                                     boolean deriveA) {
```

Comment:
Just a question here: when we will add the aibility for ratio tap changer to control voltage, we will also introduce a other boolean to say that ratio ````r1```` can variate ? It will be maybe ````deriveR1```` (and ````deriveR```` is not possible), so maybe we have to rename ````deriveA```` in ````deriveA1```` ?

Reply:
Agree it should be A1, in the code of this PR it is quite confusing.
The reason is that a2 is hardcoded to zero because never used when coming from IIDM data so a and a1 are the same everywhere (see PiModel code)
But I think we should support properly everywhere a1 and a2 variables so that if one connected to another model than IIDM where it is relevant to have a phase shift on side 2 to easily map it to OLF internal model.

Reply:
And yes, this is here that we will add r1 and r2 derive option for ratio tap changer. Maybe we should have a DeriveConfig object to avoid many parameters when adding ratio tap changers.